### DISCUSSION TITLE
How to focus panel

### DISCUSSION THREAD

**Discussion by Pyker** [jesseduffield/lazygit#3885] [General]
Hello!

This is a wonderful little tool.

I was wondering, is there a way to focus a panel? For example, when I'm looking at a commit, or when I'm staging changes, I'd like to focus panel I'm on, so it takes up the entire viewport, so I can more easily read the changes I'm making.

**Comment by Pyker** [comment]
Oh nevermind, the `+` and `_` keybinds do exactly that. Thank you!

**Reply by stefanhaller** [reply]
In addition to that, you might like the `gui.expandFocusedSidePanel` config.

**Reply by Pyker** [reply]
@stefanhaller I'll look into that too, thank you! I'll be honest, I didn't even realize lazygit had a config file until I looked through similar discussions lol
